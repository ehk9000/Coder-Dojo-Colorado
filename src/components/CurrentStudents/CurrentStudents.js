import React from 'react'

function CurrentStudents() {
  return (
    <div>
      
    </div>
  )
}

export default CurrentStudents
