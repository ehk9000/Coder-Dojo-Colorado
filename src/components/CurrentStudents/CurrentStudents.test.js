import React from 'react';
import { shallow } from 'enzyme';
import CurrentStudents from './CurrentStudents';

describe('CurrentStudents', () => {
  let wrapper;

  beforeEach(() => {
    wrapper = shallow(<CurrentStudents />);
  });

  it('should match snapshot', () => {
    expect(wrapper).toMatchSnapshot();
  });
});
