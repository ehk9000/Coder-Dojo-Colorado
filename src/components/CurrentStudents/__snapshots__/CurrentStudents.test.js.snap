// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`CurrentStudents should match snapshot 1`] = `<div />`;
